src/module/list.js

```diff
@@ -0,0 +1,32 @@
+const toDoListData = [
+  {
+    description: 'Working Out',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Cooking',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Studying',
+    completed: true,
+    index: 2,
+  },
+];
+
+const addToList = () => {
+  const listContainer = document.getElementById('listContainer');
+  for (let i = 0; i < toDoListData.length; i += 1) {
+    listContainer.innerHTML += `
+      <li class="list-to-do">
+        <input type="checkbox" class="to-do">
+        ${toDoListData[i].description}
+        <i class="fa-solid fa-trash"></i>
+      </lis>
+      <hr class="line">
+    `;
+  }
+};
```

Comment:
- [ ] Your function is displaying the todos, but not by the index order. Kindly change the display function to make the todos list appear in order of the index values as the project requires.
 - 
<img width="997" alt="image" src="https://user-images.githubusercontent.com/61919617/214908697-963198a5-b8f7-48f6-bb66-172f95a2c8d3.png">

 - [OPTIONAL] Be aware that in the next projects you will make CRUD functionalities, so it it better to make the functionalities of adding a todo to a todo array separate from the function that displays the todos from the array to the page.